text/0000-bootstrap.md

```diff
@@ -0,0 +1,68 @@
+- Start Date: 2016-05-01
+- RFC PR: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Implement a lightweight RFC process for the nix project.
+
+# Motivation
+[motivation]: #motivation
+
+Development on nix has been going well over the last few months. For the most part, adding APIs and working out how to adapt them to be more Rust-friendly has worked really well. However, there are some tougher design choices where there's been stagnation. Some examples:
+
+- string handling in nix (https://github.com/nix-rust/nix/issues/221)
+- the future of nix, including crate organization (https://github.com/nix-rust/nix/issues/190)
+
+Implementing an RFC process will give us a clear place to discuss those changes. Just as importantly, there will be a clear document stating what is being proposed. This way one doesn't need to read an entire comment thread to find out what the actual proposal is.
```

Comment:
> Implementing an RFC process will give us a clear place to discuss those changes. Just as importantly, there will be a clear document stating what is being proposed...

Simplified Wording:

The RFC process provides a place to both discuss and document major design decisions.  This provides readers with both the final decision in a clean form and (if needed) the discussion that lead up to the ratification of the RFC.